sys/arm64/arm64/exception.S

```diff
@@ -30,6 +30,7 @@
 
```

Comment:
No matter which direction things go, you absolutely need to remove the `#include <sys/intr.h>` in the second commit.

Reply:
Pointing this out twice on `arm64/exception.S` in one set of comments seems unnecessary. Maybe it was unintentional or you're not used to github's UI, but in general repeating yourself and being overly-insistent will make others less willing to engage with your suggestions (especially given that the review workflow isn't what people are used to).

Reply:
My apologies if you felt this was unnecessary.  I may not be the greatest reviewer.  I'm not at 100% and I am aware I do annoy a number of people (things which you shouldn't wish on anyone, even your enemies).